pytorch_lightning/callbacks/finetuning.py

```diff
@@ -414,3 +414,147 @@ def finetune_function(self, pl_module: 'pl.LightningModule', epoch: int, optimiz
                     f"Current lr: {round(current_lr, self.round)}, "
                     f"Backbone lr: {round(next_current_backbone_lr, self.round)}"
                 )
+
+
+class NoFreeze(BaseFinetuning):
+
+    def freeze_before_training(self, pl_module: 'pl.LightningModule') -> None:
+        pass
+
+    def finetune_function(
+        self,
+        pl_module: 'pl.LightningModule',
+        epoch: int,
+        optimizer: Optimizer,
+        opt_idx: int,
+    ) -> None:
+        pass
+
+
+class FlashBaseFinetuning(BaseFinetuning):
```

Comment:
probably not flash, right?

Reply:
yeah, that's true. gonna fix it during weekend